### DISCUSSION TITLE
zh/usage/ixah25xr/

### DISCUSSION THREAD

**Discussion by giscus** [cookieY/Yearning#1139] [FAQ]
# zh/usage/ixah25xr/

Yearning 不依赖于任何第三方 SQL 审核工具作为审核引擎,内部已自己实现审核/回滚相关逻辑。 仅依赖 Mysql 数据库。 mysql 版本必须为5.7及以上版本(8.0及以上请将sql_mode 设置为空)并已事先自行安装完毕且创建 Yearning 库,字符集应为UTF8mb4(仅 Yearning 所需 mysql 版本) Yearni...

https://next.yearning.io/zh/usage/ixah25xr/

**Comment by WallaceCan** [comment]
默认的账号密码错误了

**Reply by zc-gdgd** [reply]
使用./Yearning reset_super重置

**Comment by lufanai** [comment]
配置webhook的企微机器人链接，没有生效，手动用curl命令测机器人链接没问题

**Reply by mapsic2025** [reply]
请问下，Yearning依赖 MySQL 数据库支持8.4及以上版本吗？

目前我这边MySQL是8.0.42版本，计划升级到8.4.6版本的。

**Comment by alsww** [comment]
2.初始化及安装

**Comment by alsww** [comment]
2.初始化及安装

**Reply by alsww** [reply]
./Yearning install   , 这个 Yearning 文件不存在呢... 请问是文档没更新么?  应该是什么命令安装
